### PR TITLE
v8 fix a bunch of small issues

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
- fix issues with measuring mesh
- fix layer group reparenting issue
- fix textStyle '0' color issue
- fix tilingSprite matrix issue
- fix tiling sprite anchor issue (webGPU)

### Summary
🔄🚫➕

This pull request updates the mesh, text, and tiling sprite rendering systems to use the new WebGPU shader format and naming conventions. It also improves the performance and readability of some code by avoiding unnecessary object creation, variable confusion, and empty lines.

> _`style` and `aPosition`_
> _Renamed for clarity - kireji_
> _Winter of refactoring_

### Walkthrough
*  Rename vertex position attribute to match new shader convention ([link](https://github.com/pixijs/pixijs/pull/9488/files?diff=unified&w=0#diff-1d342e14a5fa9b47e0e30c31361244ae97e3baa7c19f0dc3af9f19750136a552L127-R127))
*  Remove default matrix value to avoid unnecessary object creation ([link](https://github.com/pixijs/pixijs/pull/9488/files?diff=unified&w=0#diff-1cd21e663358872d0530cf047d6780fa25cac642c9d729bed2ce9e8eb72ba164L28-R28))
*  Add `didChange` property to display object to track transform and geometry changes ([link](https://github.com/pixijs/pixijs/pull/9488/files?diff=unified&w=0#diff-ed831278ea825367c70b7f2674e0b5d0c3159e4cee387f2d0f674a73280beddaL141-R144))
*  Change fill style condition to handle falsy values correctly ([link](https://github.com/pixijs/pixijs/pull/9488/files?diff=unified&w=0#diff-8393c735ac09d32cf2441068d3711a155f4ef1c6aa0c82017316af90c00c486cL286-R286))
*  Rename style object variable to avoid confusion with local variable ([link](https://github.com/pixijs/pixijs/pull/9488/files?diff=unified&w=0#diff-85113c34fc7a760bb28a60f57fa23b14b107b4797230e736700f07f6dd4eab1bL251-R258))
*  Remove empty line in `tiling-sprite.wgsl` for code consistency ([link](https://github.com/pixijs/pixijs/pull/9488/files?diff=unified&w=0#diff-0b39a6696cc14b877918b64394e8046d75699831256faf6ba9def4d63f75a586L43))
*  Access texture transform matrix from tiling uniforms object in `TilingSpritePipe.ts` ([link](https://github.com/pixijs/pixijs/pull/9488/files?diff=unified&w=0#diff-50bd5bfa1f824856f8471c41605e2e3bd533518043209829c486043b1a8a249eL194-R194))

### CODE REVIEW

File: src/rendering/renderers/gl/WebGLRenderer.ts [hunk 1]
```diff
@@ -77,6 +78,8 @@ export class WebGLRenderer extends SystemManager<WebGLRenderer> implements GLRen
         });
 
         this.setup(systemConfig);
+
+        sayHello(this.type);
```
Comment (Zyie) [inline src/rendering/renderers/gl/WebGLRenderer.ts:82]: inside `StartupSystem` we already have a `hello` property that is false. So you just need to set that to true and replace the console log with `sayHello`

File: src/rendering/renderers/gpu/WebGPURenderer.ts [hunk 2]
```diff
@@ -75,6 +76,8 @@ export class WebGPURenderer extends SystemManager<WebGPURenderer> implements GPU
         });
 
         this.setup(systemConfig);
+
+        sayHello(this.type);
```
Comment (Zyie) [inline src/rendering/renderers/gpu/WebGPURenderer.ts:80]: Same as above

File: src/rendering/renderers/shared/View.ts [hunk 3]
```diff
@@ -28,6 +27,6 @@ export interface View
     addBounds: (bounds: Bounds) => void;
     containsPoint: (point: Point) => boolean;
 
-    destroy<DESTROY_OPTIONS = DestroyOptions>(options: DESTROY_OPTIONS): void;
+    destroy(options?: unknown): void;
```
Comment (Zyie) [inline src/rendering/renderers/shared/View.ts:30]: This should still be `DestroyOptions`?

Comment (GoodBoyDigital) [reply src/rendering/renderers/shared/View.ts:30]: changed!

File: src/rendering/renderers/gl/WebGLRenderer.ts (hunk 1 above)
Comment (GoodBoyDigital) [reply src/rendering/renderers/gl/WebGLRenderer.ts:82]: ah yes!

File: src/maths/shapes/Ellipse.ts [hunk 4]
```diff
@@ -111,7 +111,7 @@ export class Ellipse implements ShapePrimitive
     // #if _DEBUG
     public toString(): string
     {
-        return `[@pixi/math:Ellipse x=${this.x} y=${this.y} width=${this.width} height=${this.height}]`;
+        return `[@pixi/math:Ellipse x=${this.x} y=${this.y} width=${this.halfWidth} height=${this.halfHeight}]`;
```
Comment (Zyie) [inline src/maths/shapes/Ellipse.ts:114]: If this has changed to half width/height then this log should also say halfWidth=...

Comment (GoodBoyDigital) [reply src/maths/shapes/Ellipse.ts]: good catch! updated

**Review by Zyie — APPROVED** [review]

**Pull request merged**